webstrom/urls.py

```diff
@@ -3,19 +3,24 @@
 from django.contrib import admin
 from django.urls import include, path
 
-urlpatterns = [
-    path('django-admin/', admin.site.urls),
+api_urlpatterns = [
     path('user/', include('user.urls')),
     path('competition/', include('competition.urls')),
     path('cms/', include('cms.urls')),
     path('personal/', include('personal.urls')),
     path('base/', include('base.urls')),
     path('protected/', include('downloads.urls')),
-    # Dočasná cesta pre allauth bez rest frameworku
+]
+
+urlpatterns = [
     path('accounts/', include('allauth.urls')),
+    path('django-admin/', admin.site.urls),
```

Comment:
rozmyslam.. predpokladam, ze toto mame tiez v nginx ako routu. tak ci next rewrity/middleware nedrzat naozaj in sync s nginxom - napr. lokalne tym padom mozeme spravit `localhost:3000/django-admin` a smerovat to bude na BE (podla toho, ktory aktualne pouzivame). v podstate aby FE podporoval vsetky routy, ktore podporuje deployed stranka, aj lokalne.

posli tym padom prosim niekam vycet vsetkeho, co v nginx mame definovane

Reply:
Aktuálne ten nginx config vyzerá takto
```
server {
    location /static/ {
        proxy_pass http://localhost:8921/;
    }

    location /media/ {
        alias /data/www/webstrom-test/media/;
    }

    location /django-admin/ {
        proxy_set_header X-Forwarded-Host $host;
        proxy_set_header X-Forwarded-Proto https;

        proxy_pass http://localhost:8920/django-admin/;
    }

    location /api/ {
    	proxy_set_header X-Forwarded-Host $host;
	proxy_set_header X-Forwarded-Proto https;

	proxy_pass http://localhost:8920/api/;
    }

    location / {
        proxy_set_header X-Forwarded-Host $host;
        proxy_set_header X-Forwarded-Proto https;

        proxy_pass http://localhost:8922/;
    }
}
```

Neviem či musím nutne držať tie rewrity 1:1 s nginxom, asi mi skôr príde lepšie ich obmedziť len na to čo je nutné aby ten frontend sa dal vyvíjať a zvyšok si proste vždy vieme otvoriť na backende. Ale nemám na to nejaký silný názor.